pypit/ardebug.py

```diff
@@ -1,8 +1,11 @@
 from __future__ import (print_function, absolute_import, division, unicode_literals)
 
+import matplotlib.pyplot as plt
+import numpy as np
+
 # These need to be outside of the def's
-from pypit.ginga import show_image
-from pypit.ginga import chk_arc_tilts
+#from pypit.ginga import show_image
```

Comment:
Is this a circular imports issue?

These changes would reduce the ease-of-use
of this debugger..

Let's discuss.

Reply:
I think I commented these out because they're not actually used in the module itself.  Are you importing them here because they get used when debugger is imported?

Reply:
yes, that is the intent.

Reply:
Ah, got it.  I just saw how this is used in artrace.py .  We can leave it that way if you want, but I'm not crazy about the construction.  E.g., why are ginga.py and ardebug.py separate modules?  In the one place chk_arc_tilts() is used (artrace.py) the module actually imports ginga as well (although that's probably because I moved all the imports to the top), so why not use it directly from ginga?